lua/treesitter-context.lua

```diff
@@ -13,6 +13,7 @@ end
 local defaultConfig = {
   enable = true,
   max_lines = 0, -- no limit
+  zindex = 40,
```

Comment:
I would consider even lower. Say 20 or 30? Is there anything that would ever go below context?